### PR TITLE
6115 - Implement HoloFuel MVP v.0.1.3 UI design - Inbox

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: src/holofuel/pages/Inbox/Inbox.js [hunk 1]
```diff
@@ -162,35 +158,34 @@ export default function Inbox () {
       title={displayBalance}
       titleSuperscript='Balance'
     >
-      <Button styleName='new-transaction-button' onClick={() => showNewTransactionModal()}>
-        <AddIcon styleName='add-icon' color='#0DC39F' />
-        <h3 styleName='button-text'>New Transaction</h3>
+      <Button styleName='new-transaction-button'
+        variant='green'
+        onClick={() => push(OFFER_REQUEST_PATH)}>
```
Comment (JettTech) [inline src/holofuel/pages/Inbox/Inbox.js:163]: Nice update. :+1:

File: src/holofuel/pages/Inbox/Inbox.js [hunk 2]
```diff
@@ -353,23 +346,6 @@ function DeclineOrCancelButton ({ showConfirmationModal, transaction, isDeclined
   </Button>
 }
 
-function NewTransactionModal ({ handleClose, isNewTransactionModalVisible }) {
```
Comment (JettTech) [inline src/holofuel/pages/Inbox/Inbox.js:356]: Glad we can take this one out of here. :100:

File: src/holofuel/pages/Inbox/Inbox.js [hunk 3]
```diff
@@ -398,19 +374,19 @@ export function ConfirmationModal ({ transaction, handleClose, declineTransactio
       contentLabel = 'Pay request'
       actionParams = { id, amount, counterparty }
       actionHook = payTransaction
-      message = <div styleName='modal-text' data-testid='modal-message'>
```
Comment (JettTech) [inline src/holofuel/pages/Inbox/Inbox.js:401]: I don't see any updates to the tests that would adjust to this change.  Are not we verifying the message is correct in the tests?

**Review by JettTech — APPROVED** [review]
Overall, this is ready to ship. I have one question that is non-blocking, but worth a small discussion. :shipit:

File: src/holofuel/pages/Inbox/Inbox.js (hunk 3 above)
Comment (robbiecarlton) [reply src/holofuel/pages/Inbox/Inbox.js:401]: This doesn't change the message, it just moves the styling and data id out of this variable and into the main component jsx

**Pull request merged**